include/extractor/guidance/turn_classification.hpp

```diff
@@ -25,7 +25,7 @@ namespace guidance
 {
 
 std::pair<util::guidance::EntryClass, util::guidance::BearingClass>
-classifyIntersection(Intersection intersection);
+classifyIntersection(ConnectedRoads intersection);
```

Comment:
Same here, if we name it roads, the connection between `intersection` and `connected roads` is not clear.

Sure, a `vector<ConnectedRoad>` is a list of `ConnectedRoads`, but so is `std::set<ConnectedRoad>` or `std::list<ConnectedRoad>`. I don't really like removing the additional information that is inherently given by `having a specifically sorted list of Roads`. Especially with the added functionality.

It's a dedicated class, not simply a plain collection.